docs/guides/integration-mobile-client-side.md

```diff
@@ -0,0 +1,585 @@
+
+The first time a new user opens the app, no EUID identity exists. You'll need to call the `generateIdentity` method, with the personal data, to start the token generation:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+EUIDManager.getInstance().generateIdentity(
+    identityRequest: IdentityRequest,
+    subscriptionId: String,
+    publicKey: String,
+    onResult: (GenerateIdentityResult) -> Unit
+)
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+EUIDManager.shared.generateIdentity(
+    _ identity: IdentityType,
+    subscriptionID: String,
+    serverPublicKey: String,
+    appName: String? = nil
+)
+```
+
+</TabItem>
+</Tabs>
+
+When this method call completes successfully, the advertising token (EUID token) is available for you to send to the bidstream.
+
+If the EUID identity stored in local file storage has expired and cannot be refreshed, you must call the `generateIdentity` method again to generate a new identity and get the resulting EUID token. The only exception is when the response to the following Android method/iOS object indicates that the personal data was opted out of EUID:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+**Android Java**:
+
+```java
+EUIDManager.getInstance().getCurrentIdentityStatus()
+```
+
+**Android Kotlin**:
+
+```kotlin
+EUIDManager.getInstance().currentIdentityStatus()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+EUIDManager.shared.identityStatus
+```
+
+</TabItem>
+</Tabs>
+
+A response status of `OPT_OUT` for Android, or `optOut` for iOS, indicates that the personal data has been opted out of EUID and no identity/token should be generated for it. You might want to avoid making repeated `generateIdentity` calls: if the personal data has a status of opted out, the EUID token is not generated.
+
+The best way to determine if personal data is required by the EUID mobile SDKs is to always call the `getAdvertisingToken()` method when the app starts up or resumes:
+
+<Tabs groupId="language-selection">
+<TabItem value='android' label='Android'>
+
+```js
+EUIDManager.getInstance().getAdvertisingToken()
+```
+
+</TabItem>
+<TabItem value='ios' label='iOS'>
+
+```js
+EUIDManager.shared.getAdvertisingToken()
+```
+
+</TabItem>
+</Tabs>
+
+If `getAdvertisingToken()` returns null, and the identity status is not `OPT_OUT`/`optOut`, you'll need to generate a new token. To do this, pass the personal data into the `generateIdentity` method again. For details, see [Configure the EUID mobile SDK](#configure-the-euid-mobile-sdk).
+
+## Enable Logging
+
+<EnableLogging />
+
+## Optional: EUID GMA/IMA Plugin for GAM Secure Signals integration
+
+<GMAIMA_Plugins />
+
+
+## Optional: EUID Prebid Mobile SDK Integration
+
+:::important
+The EUID Prebid Mobile SDK integration requires version 1.4.0 of the EUID SDK for Android, or version 1.5.0 of the EUID SDK for iOS.
```

Comment:
Let's update the version. The IOS sdk version is 1.7.0 and android sdk version is 1.5.0

Reply:
Done.